resources/views/pages/achievement/[achievement]/report-issue.blade.php

```diff
@@ -98,64 +98,88 @@
+        <x-ticket.guide-link
+            buttonText="Create Ticket"
+            href="{{ route('achievement.create-ticket', ['achievement' => $achievement]) }}?type={{ TicketType::TriggeredAtWrongTime }}"
+        >
             I unlocked this achievement without meeting the requirements, and then I reset it.
         </x-ticket.guide-link>
 
         @php
-            $subject = urlencode("Manual Unlock: $achievement->title ({$achievement->game->title})");
-            $message = urlencode("I'd like a manual unlock for [ach=$achievement->id]:\n(provide link to video/screenshot showing evidence)");
+            $subject = urlencode("Manual Unlock: {$achievement->title} [{$achievement->id}] ({$achievement->game->title})");
+            $message = urlencode("I'd like a manual unlock for [ach={$achievement->id}]:\n(provide link to video/screenshot showing evidence)");
         @endphp
-        <x-ticket.guide-link buttonText="Request Manual Unlock"
-                             href="{{ route('message.create') }}?to=RAdmin&subject={{ $subject }}&message={{ $message }}">
+        <x-ticket.guide-link
+            buttonText="Request Manual Unlock"
+            href="{{ route('message.create') }}?to=RAdmin&subject={{ $subject }}&message={{ $message }}&templateKind='manual-unlock'"
+        >
             The achievement triggered, but the unlock didn't appear on my profile.
         </x-ticket.guide-link>
     @else
-        <x-ticket.guide-link buttonText="Create Ticket"
-                             href="{{ route('achievement.create-ticket', ['achievement' => $achievement]) }}?type={{ TicketType::TriggeredAtWrongTime }}">
+        <x-ticket.guide-link
+            buttonText="Create Ticket"
+            href="{{ route('achievement.create-ticket', ['achievement' => $achievement]) }}?type={{ TicketType::TriggeredAtWrongTime }}"
+        >
             I unlocked this achievement without meeting the requirements.
         </x-ticket.guide-link>
     @endif
 
     @php
-        $subject = urlencode("Typo: $achievement->title ({$achievement->game->title})");
-        $message = urlencode("I'd like to report a spelling/grammar error in [ach=$achievement->id]:\n(Describe the issue here)");
+        $subject = urlencode("Typo: {$achievement->title} [{$achievement->id}] ({$achievement->game->title})");
+        $message = urlencode("I'd like to report a spelling/grammar error in [ach={$achievement->id}]:\n(Describe the issue here)");
     @endphp
-    <x-ticket.guide-link buttonText="Message WritingTeam"
-                         href="{{ route('message.create') }}?to=WritingTeam&subject={{ $subject }}&message={{ $message }}">
+    <x-ticket.guide-link
+        buttonText="Message WritingTeam"
+        href="{{ route('message.create') }}?to=WritingTeam&subject={{ $subject }}&message={{ $message }}&templateKind='writing-error'"
```

Comment:
Are the quotes intentional on `templateKind`?

Also occurs on other links

Reply:
Nope - this was an accident, but still worked all the same. Removed them in latest.